docs/source/conf.py

```diff
@@ -258,8 +237,8 @@
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
-  (master_doc, 'ibmdbpy.tex', 'ibmdbpy Documentation',
-   'Edouard Fouché', 'manual'),
+    (master_doc, 'ibmdbpy.tex', u'ibmdbpy Documentation',
+     u'Edouard Fouché,Avipsa Roy', 'manual'),
```

Comment:
I guess you don't need the u'' format. Strings should be ok. Plus, we do Python3.